cmd/apic/template.go

```diff
@@ -42,11 +43,12 @@ func init() {
 }
 
 type templateVerb struct {
-	Dir        string        `help:"The output directory"`
-	Tracer     string        `help:"The template function trace expression"`
-	Gopath     string        `help:"the go path to use when looking up packages"`
-	GlobalList []string      `help:"A global value setting for the template"`
-	Search     file.PathList `help:"The set of paths to search for includes"`
+	Dir            string        `help:"The output directory"`
+	Tracer         string        `help:"The template function trace expression"`
+	Gopath         string        `help:"the go path to use when looking up packages"`
+	GlobalList     []string      `help:"A global value setting for the template"`
+	Search         file.PathList `help:"The set of paths to search for includes"`
```

Comment:
rename variable `Search` to `IncludeSearch`, now that we also have a `TemplateSearch`

Reply:
I don't want to change IncludeSearch, as it breaks the public interface of this program. We have down-stream folks using this.

Reply:
Makes sense. Maybe document the fact that this interface is public, and should not be changed lightheartedly?